android/app/src/main/java/com/expensify/chat/intentHandler/ShareFileObject.kt

```diff
@@ -0,0 +1,9 @@
+package com.expensify.chat.intentHandler
+
+import com.google.gson.Gson
+
+data class ShareFileObject(val content: String, val mimeType: String?) {
+    override fun toString(): String {
+        return Gson().toJson(this)
```

Comment:
Do we need to add library for handling json for this line? Wouldn't it be easier just to do something like:
```kotlin
    override fun toString(): String {
        return "{\"content\": ${content}, \"mimeType\": ${mimeType}}"
```